skimage/util/montage.py

```diff
@@ -6,7 +6,7 @@
 EPSILON = 1e-6
 
 
-def montage2d(arr_in, fill='mean', rescale_intensity=False, grid_shape=None):
```

Comment:
I'd rename this parameter as 'padding_width' or smth like that. `border_padding` sounds like something binary (yes/no).